tests/inputFields.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  //1. Select the PET TYPES menu item in the navigation bar
+  await page.getByText('Pet Types').click()
+  //2. Add assertion of the "Pet Types" text displayed above the table with the list of pet types
+  expect(await page.locator('h2').getByText('Pet Types').isVisible()).toBeTruthy()
```

Comment:
While this type of the assertion technically is working, it's not the best way to validate if some particular text is expected on the page. The main reason - the error message that you see if the test fails. It would look like this:
![image](https://github.com/b-w-cole/playwright-homework/assets/155187576/f7212d45-83ea-4c15-a583-8c5e59e90472)

As you can see, the message is completely misleading. It's not telling what exactly is wrong.
What you would like to have is message like this:
![image](https://github.com/b-w-cole/playwright-homework/assets/155187576/f4ce5a7a-ef8f-42da-b682-4f672064df73)

Modify all similar assertions in your code to have a style like this to be more meaningful